prime-angular-frontend/src/app/modules/enrolment/shared/guards/enrolment.guard.ts

```diff
@@ -45,6 +45,15 @@ export class EnrolmentGuard implements CanActivate, CanActivateChild, CanLoad {
     return this.canActivate(next, state);
   }
 
+  public navigate(routePath: string, navPath: string) {
```

Comment:
Guards provide a limited API of usable methods for the router so the navigate method could be private